solr/core/src/test/org/apache/solr/cli/AuthToolTest.java

```diff
@@ -70,7 +70,7 @@ public void testEnableAuth() throws Exception {
       dir.toAbsolutePath().toString(),
       "--solr-include-file",
       solrIncludeFile.toAbsolutePath().toString(),
-      "-credentials",
```

Comment:
so, @gerlowskija  and I had a conversation that we ideally should be randomizing some of these, like `-credentials` and `--credentials` to make sure things continue to work with deprecated and non deprecated options.   I took the short cut of just randomly picking between the two, however maybe we should introduce a method here to handle this?  Alternatively, maybe a comment on why `-credentials` just to remind the next person?

Reply:
Here was the ticket for tracking the randomization idea: https://issues.apache.org/jira/browse/SOLR-17344

Reply:
after dinking around with the code, I realize that the `-` versus `--` is dealt with by commons-cli, and we don't need to test that logic!   I will look for a place to introduce testing of deprecated Option versus current Option...